Jenkinsfile

```diff
@@ -33,8 +33,8 @@ timestampedNode('java') {
     timeout(time: 180, unit: 'MINUTES') {
         // See below for what this method does - we're passing an arbitrary environment
         // variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
-        withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m",
-                      "MAVEN_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m"]) {
+        withMavenEnv(["JAVA_OPTS=-Xmx2g -Xms512m -XX:MaxPermSize=1024m",
```

Comment:
as it was the tests that fail is this really needed?

From the build console log

> 13:13:55 [INFO] BUILD SUCCESS
13:13:55 [INFO] ------------------------------------------------------------------------
13:13:55 [INFO] Total time: 01:30 h
13:13:55 [INFO] Finished at: 2016-02-16T18:13:55+00:00
13:13:56 [INFO] Final Memory: 171M/721M